README.md

```diff
@@ -0,0 +1,59 @@
+# Role Name
+
+This is an ansible role which sets up a php-fpm instance and configures the pools.
+
+
+## Requirements
+
+This role needs systemd and an apt based package manager.
+
+
+## Role Variables
+
+To see all vars possible for php-fpm see 
+[php-fpm config page](https://secure.php.net/manual/en/install.fpm.configuration.php)
+
+
+To see all vars possible for php.ini see
+[php.ini page](https://secure.php.net/manual/de/ini.list.php)
+
+```yml
+fpm_conf.log_level: error
+php_ini_values:
```

Comment:
duplicate of the example below. what is this supposed to be?
Please explain your basic variable structure and list at least the mandatory vars.